packages/remirror__extension-code-block/package.json

```diff
@@ -59,19 +59,14 @@
     "@remirror/cli": "1.1.0-beta.1",
     "@remirror/pm": "3.0.0-beta.5",
     "@types/jsdom": "^16.2.15",
-    "@types/prettier": "^2.7.2",
     "jsdom": "^17.0.0",
-    "prettier": "^2.8.8"
+    "prettier": "^3.2.5"
   },
   "peerDependencies": {
     "@remirror/pm": "^3.0.0-beta.5",
-    "@types/prettier": "^2.7.2",
-    "prettier": "^2.8.8"
+    "prettier": "^3.2.0"
   },
   "peerDependenciesMeta": {
-    "@types/prettier": {
-      "optional": true
-    },
```

Comment:
I think this peer dep should still be optional for 2 reasons

As we have a `formatter` option here, the user might be using something _other_ than prettier.  

I suspect this might also result needing to have prettier installed, whether you use the code block extension or not.

Reply:
I'm not sure I follow. I've only removed `@types/prettier` here because types are included in prettier v3. `prettier` remains listed as an optional peer dependency. Do I misunderstand something?

Reply:
Ugh, my mistake. I forgot about the type being included with v3 thing.

I need more sleep clearly! 😴